src/include/network/routines.rb

```diff
@@ -739,6 +745,32 @@ def SetAllLinksDown
       interfaces.all? { |i| SetLinkDown(i) }
     end
 
+    # Calls wicked ifdown if it is the currenct backend, set the link down
```

Comment:
there should be stated a difference between "ip set link down" and "wicked down" (the other resets wicked's cache)